scripts/PMID_downloader.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from Bio import Entrez
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
```

Comment:
Seems like you added two additional packages to our setup.
Could you add them to the requirements_scripts.txt, with version number please.